### PR TITLE
Ease of use Updates

### PR DESCRIPTION
**Pull request by ablasdel**

### CODE REVIEW

File: source/in_the_box.rst [hunk 1]
```diff
@@ -107,20 +108,25 @@ and network is setup for multicast DNS (mDNS) then you may be able to use
...
 .. warning::
     Monitor cables that conform to HDMI specifications will fit the robot's monitor port. But some cheaper cables have overmolding that is too large given the spec and may not seat properly. Please ensure the HDMI cable you are using is the correct size.
 
+.. note::
+
+   If you are still having difficulty connecting to your robot you will need to contact your
+   network administrator for specific instructions on how to connect the robot to your network.
+
+
+Once Connected then create your user account as shown below.
```
Comment (erelson) [inline source/in_the_box.rst]: s/Connected/connected

File: source/in_the_box.rst [hunk 2]
```diff
@@ -107,20 +108,25 @@ and network is setup for multicast DNS (mDNS) then you may be able to use
...
-
-.. include:: computer.rst
-   :start-after: embed-user-accounts-start
-   :end-before: embed-user-accounts-end
-
-Connecting a Monitor
+Connecting a Monitor (Optional)
 ~~~~~~~~~~~~~~~~~~~
-If you are unable to access the robot through ssh due to your network settings you will need to connect an HDMI monitor, USB keyboard and USB mouse to the side panel ports. Then use them to select your network as you would a normal Ubuntu computer. Once it is set it should remain connected through reboots.
+If you are unable to access the robot through ssh due to your network settings you will need to connect an HDMI monitor, USB keyboard and USB mouse to the side panel ports. Then use them to select your network from the networking menu, it looks like an empty quarter circle pointed downward when disconnected, in the upper right hand corner of the screen. Once it is set it should remain connected through all reboots.
```
Comment (mikeferguson) [inline source/in_the_box.rst]: while editing this, we should wrap this line at 80 chars for readability of future changes

File: source/teleop.rst [hunk 3]
```diff
@@ -82,5 +89,6 @@ package.
 
 ::
 
+    >$ export ROS_MASTER_URI=http://<robot_name_or_ip>:11311
```
Comment (mikeferguson) [inline source/teleop.rst]: this doesn't seem to have the same indentation as the next line

Comment (ablasdel) [inline source/teleop.rst]: Looks like the old one was a tab. converted it over.

**Comment by ablasdel** [comment]
Fixed noted issues.

File: source/teleop.rst [hunk 4]
```diff
@@ -82,5 +89,6 @@ package.
 
 ::
 
-	>$ rosrun teleop_twist_keyboard teleop_twist_keyboard.py
+  >$ export ROS_MASTER_URI=http://<robot_name_or_ip>:11311
```
Comment (mikeferguson) [inline source/teleop.rst]: these appear to be 2 spaces -- if I recall, they need to be at least 3 for the sphinx to render them as indented properly

**Comment by mikeferguson** [comment]
now sync'd to public

**Pull request merged**